openedx/core/djangoapps/content_libraries/views.py

```diff
@@ -181,14 +178,16 @@ class LibraryApiPagination(PageNumberPagination):
 
 @method_decorator(non_atomic_requests, name="dispatch")
 @view_auth_classes()
-class LibraryRootView(APIView):
+class LibraryRootView(GenericAPIView):
     """
     Views to list, search for, and create content libraries.
     """
 
+    _DEFAULT_PAGE_SIZE = 50
```

Comment:
Good point, I had it there initially mainly because it was already there before and I didn't want to change the existing functionality. I've removed it here [53c8a39](https://github.com/openedx/edx-platform/pull/34879/commits/53c8a39800b86d0ef28b4e1d9dc756126a7eab2c)